### DISCUSSION TITLE
CC: Tweaked 1.105 for MC 1.4.7, 1.19.4 and 1.20

### DISCUSSION THREAD

**Discussion by SquidDev** [cc-tweaked/CC-Tweaked#1482] [Announcements]
Hello again! Last week Minecraft 1.20 was released, with all sorts of fancy new features (and optimisations), and this means it's for a new CC: Tweaked release!

As is often the case, there's not any spectacular changes (sorry!), but there's still plenty of small and useful improvements.

# Minecraft updates
As you can probably predict, CC: Tweaked has been updated to Minecraft 1.20 (and subsequently 1.20.1). There has been a small regression with off-hand pocket computers (https://github.com/cc-tweaked/CC-Tweaked/issues/1471), but everything else appears to be pretty stable. Of course, please do report any issues you encounter!

Perhaps less predictably, CC: Tweaked has also been released for Minecraft 1.4.7. [This release](https://modrinth.com/mod/cc-tweaked/version/IPsqMBvF) is a core mod which sits alongside ComputerCraft 1.50, backporting the latest Cobalt and CraftOS:

![A screenshot of a ComputerCraft terminal, running in Minecraft 1.4.7. It is showing off the new error messages and box-drawing characters - features which most definitely did not exist back then!](https://github.com/cc-tweaked/CC-Tweaked/assets/4346137/2191fa3e-f002-45c6-b65f-e305ea2e6632)

I'd recommend running this alongside RedPower 2 and [asie's rpcDrive](https://modrinth.com/mod/rpcdrive), for the ultimate inauthentic 1.4.7 experience.

Minecraft 1.19.4 will continue to our main development version - don't worry, we're not dropping support for it!

# Other highlights:
 - The maximum size of files which can be uploaded by drag-and-drop can now be set via the config file. Thank you @khankul!
 - @Lemmmy added proxy support to the HTTP API, closing one of CC:T's oldest issues.

---

As always, thank you for everyone who has contributed to this release!

**Comment by fatboychummy** [comment]
> CC: Tweaked has also been released for Minecraft 1.4.7

![you what](https://media.tenor.com/3-Pm8cYrdqEAAAAC/spongebob-you-what.gif)

**Reply by fatboychummy** [reply]
This is entirely unexpected, haha.

I'm really curious what caused you to decide to backport CC? Just testing things for the hell of it, or got a modpack you wanted to play on an older version of MC? (Something else?)

**Reply by SquidDev** [reply]
Got nerd sniped.

**Comment by ManDongI** [comment]
Hello. Is 1.20 version of cc tweaked published on maven?

**Reply by piprett** [reply]
Yes. [Link: squiddev.cc](https://squiddev.cc/maven/cc/tweaked/)
![image](https://github.com/cc-tweaked/CC-Tweaked/assets/49862976/50cb1ed0-1e23-4317-9f1c-647fc8e81784)
